.github/workflows/scala.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         distribution: 'temurin'
         cache: 'sbt'
     - name: Run tests
-      run: sbt test
+      run: sbt +test
```

Comment:
Here I made sure the project is cross tests against Scala 2.13 and Scala 3.
Please make sure you cross publish when releasing, so there are artifacts for both:
- Scala 2.13: https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_2.13/
- and Scala 3: https://repo1.maven.org/maven2/com/github/fdimuccio/play2-sockjs_3/ (does not exist yet of course)

Thanks!